project/CompilerSettings.scala

```diff
@@ -26,10 +26,8 @@ object CompilerSettings {
       "-feature",
       "-language:_",
       "-unchecked",
-      /*
-       * With > 16: [error] invalid setting for -Ybackend-parallelism must be between 1 and 16
-       * https://github.com/scala/scala/blob/v2.12.6/src/compiler/scala/tools/nsc/settings/ScalaSettings.scala#L240
-       */
+      //With > 16: [error] invalid setting for -Ybackend-parallelism must be between 1 and 16
+      //https://github.com/scala/scala/blob/v2.12.6/src/compiler/scala/tools/nsc/settings/ScalaSettings.scala#L240
```

Comment:
Block comment changed to line comment

Reply:
This I changed myself, the formatted version was ugly otherwise.